File: Algo_Java/Exercices/src/Exercices/Exo1_5.java
package Exercices;
import java.util.Scanner;

public class Exo1_5 {
	public static void main(String args[]) {
		int a;
		int b;
		int temp;
		
		Scanner scanner;
		scanner = new Scanner(System.in);
		
		System.out.println("Programme d'échange entre une valeur A et B.");
		System.out.println("Saisissez votre valeur A: ");
		a = scanner.nextInt();
		System.out.println("Saisissez votre valeur B: ");
		b = scanner.nextInt();
		System.out.println("Votre valeur A est " + a + " et votre valeur B est " + b);
		temp = a;
		a = b;
		b = temp;
		System.out.println("Votre valeur A est maintenant " + a + " et votre valeur B est maintenant " + b);
		
		scanner.close();
	}
}

File: Python/Exercices/Exo1_5.py
#coding:utf-8

a = int(input("Saisir valeur A : "))
b = int(input("Saisir valeur B : "))
print("Votre valeur A est", a, "et votre valeur B est", b)
temp = a
a = b
b = temp
print("Votre valeur A est maintenant", a, "et votre valeur B est maintenant", b)